src/coreclr/debug/daccess/request.cpp

```diff
@@ -4156,12 +4156,15 @@ TADDR ClrDataAccess::DACGetManagedObjectWrapperFromCCW(CLRDATA_ADDRESS ccwPtr)
     return managedObjectWrapperPtr;
 }
 
-HRESULT ClrDataAccess::DACTryGetComWrappersObjectFromCCW(CLRDATA_ADDRESS ccwPtr, OBJECTREF* objRef)
+HRESULT ClrDataAccess::DACTryGetComWrappersHandleFromCCW(CLRDATA_ADDRESS ccwPtr, OBJECTHANDLE* objHandle)
 {
-    if (ccwPtr == 0 || objRef == NULL)
-        return E_INVALIDARG;
+    HRESULT hr = E_FAIL;
 
-    SOSDacEnter();
```

Comment:
Did you drop the SOSDacEnter on purpose?

Reply:
The macos build is complaining about gotos jumping over variable initialization, it's probably from getting rid of the try/catch in SOSDacEnter

Reply:
I don't think function needs the SOSDacEnter/Leave because it is only called internally by other functions that do or it is called by DBI via the dbi/dac API.  

The Linux/MacOS build errors because of the goto's need to be fixed.